.github/actions/custom-actions/aselo_staging_custom/action.yml

```diff
@@ -0,0 +1,48 @@
+name: 'Aselo Beta release custom action'
+description: 'Adds Twitter related environment variables to the Aselo Beta environment'
```

Comment:
I think the description should be something more general. Is true that _for now_ we use this for Twitter creds only, but if we ever want Instagram in Beta, this same action will be extended.